site/src/js/library/modules/callbacks/NewsDetailWidget.js

```diff
@@ -1,7 +1,9 @@
 import { nciProfileInfo } from "../helpers/NciProfileInfo";
+import { formatting } from "../helpers/Formatting";
 
-const newsDetail_ui_prepContacts = function (widget, _props) {
-  nciProfileInfo.updateProfileLinks(widget)
+const newsDetail_ui_prepContacts = function (widget, data, _props) {
```

Comment:
This is the ui callback, which means what gets passed in won't actually be useful. 

We'll need to do a new `newsDetail_data_changeDate` callback. That would get put into the config here:

https://github.com/NCIOCPL/akumina-mynci/blob/000d757454828f2cf02b2e4e911af7f198504144/site/src/js/widgets/NewsDetailWidget/config/config.json#L69

Reply:
Make sure that also gets added to the exports:

https://github.com/NCIOCPL/akumina-mynci/blob/000d757454828f2cf02b2e4e911af7f198504144/site/src/js/library/modules/callbacks/NewsDetailWidget.js#L8

Reply:
I did just verify that for newsDetailWidget the field is still `data.newsarticle.Publish_x0020_Date`.